source-to-image/buildah.Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM centos:7 as base
```

Comment:
Can we just share this with the buildah builder?

https://github.com/knative/build-templates/blob/master/buildah/Dockerfile

Reply:
yes definitely :wink:

Reply:
how best to reuse that Dockerfile? symlink?